airflow/ui/src/App.tsx

```diff
@@ -19,12 +19,43 @@
 
 import { hot } from 'react-hot-loader';
 import React from 'react';
-import { Center, Heading } from '@chakra-ui/react';
+import { Route, Redirect, Switch } from 'react-router-dom';
+
+import Pipelines from 'views/Pipelines';
+import Pipeline from 'views/Pipeline';
+
+import EventLogs from 'views/Activity/EventLogs';
+
+import Config from 'views/Config';
+
+import Access from 'views/Access';
+import Users from 'views/Access/Users';
+import Roles from 'views/Access/Roles';
+
+import Docs from 'views/Docs';
+import NotFound from 'views/NotFound';
 
 const App = () => (
-  <Center height="100vh">
-    <Heading>Apache Airflow new UI</Heading>
-  </Center>
+  <Switch>
+    <Redirect exact path="/" to="/pipelines" />
+    <Route exact path="/pipelines" component={Pipelines} />
+    <Route exact path="/pipelines/:dagId" component={Pipeline} />
+
+    <Route exact path="/activity/event-logs" component={EventLogs} />
+
+    <Route exact path="/config" component={Config} />
+
+    <Route exact path="/access" component={Access} />
+    <Route exact path="/access/users" component={Users} />
+    <Route exact path="/access/users/new" component={Users} />
+    <Route exact path="/access/users/:id" component={Users} />
```

Comment:
Should we use `username` instead of `Id`?

Reply:
Are usernames unique? If so, I'd say yes.

Reply:
I'm wondering why we're using username instead of ID for the API endpoint if it's not immutable?

Reply:
> Are usernames unique? If so, I'd say yes.

Yes, It's unique

> I'm wondering why we're using username instead of ID for the API endpoint if it's not immutable?

It'll be easier to remember than the ID that's why I prefer it